src/cli/limeTRX.cpp

```diff
@@ -276,186 +229,95 @@ class ConstellationPlotter
 };
 #endif
 
-static std::vector<int> ParseIntArray(std::string_view str)
+static std::vector<int> ParseIntArray(args::NargsValueFlag<int>& flag)
 {
     std::vector<int> numbers;
-    while (str.length() > 0)
-    {
-        try
-        {
-            std::size_t index = str.find_first_of(',');
-            int nr = std::stoi(std::string{ str.substr(0, index) });
-            numbers.push_back(nr);
-
-            if (index == std::string_view::npos)
-            {
-                return numbers;
-            }
-
-            str = str.substr(index + 1);
-        } catch (...)
-        {
-            return numbers;
-        }
-    }
+    for (const auto& number : args::get(flag))
+        numbers.push_back(number);
     return numbers;
 }
 
 int main(int argc, char** argv)
 {
-    StreamComposite* composite = nullptr;
-    std::string_view devName{ ""sv };
-    std::filesystem::path rxFilename{ ""sv };
-    std::filesystem::path txFilename{ ""sv };
-    bool rx = true;
-    bool tx = false;
-    bool showFFT = false;
+    // clang-format off
+    args::ArgumentParser                parser("limeTRX - Realtime streaming of RF samples", "");
+    args::HelpFlag                      helpFlag(parser, "help", "This help", {'h', "help"});
+
+    args::ValueFlag<std::string>        deviceFlag(parser, "name", "Specifies which device to use", {'d', "device"});
+    args::NargsValueFlag<int>           chipFlag(parser, "index", "Specify chip index, or index list for aggregation [0,1...]", {'c', "chip"}, {1, static_cast<size_t>(-1)});
```

Comment:
It's unclear what `{1, static_cast<size_t>(-1)}` is, is it default value, or accaptable range?